README.md

```diff
@@ -1,64 +1,6 @@
 [![official project](http://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
-[![version](https://img.shields.io/badge/dynamic/json.svg?color=orange&label=latest%20version&query=%24.tag_name&url=https%3A%2F%2Fgithub.com%2FJetBrains%2Fkotlin-native%2Freleases%2Flatest)](https://github.com/JetBrains/kotlin-native/releases/latest)
 # Kotlin/Native  #
 
-_Kotlin/Native_ is an LLVM backend for the Kotlin compiler, runtime
-implementation, and native code generation facility using the LLVM toolchain.
+This repository is obsolete and archived.
```

Comment:
Let's not use "obsolete" here, because it could be interpreted as "Kotlin/Native is obsolete".
I propose:
> The development of Kotlin/Native was moved to https://github.com/JetBrains/kotlin. Because of that, this repository was archived.